ckanext/issues/model/__init__.py

```diff
@@ -391,13 +391,30 @@ def get_comment_count(cls, issue):
             filter(cls.issue_id == issue.id).count()
 
     @classmethod
-    def reported_comments(cls, session, organization_id):
-        return session.query(IssueComment, Issue) \
+    def reported_comments(cls, session, organization_id=None):
+        query = session.query(IssueComment, Issue) \
             .join(Issue) \
             .join(model.Package) \
             .filter(cls.visibility == u'hidden') \
```

Comment:
Actually I spotted that this is wrong. Visibility only goes to hidden after a configurable number of reports. We need to find out the reported comments using a different query.